migrations/20200824143059-new-order-statuses.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      `UPDATE "Orders"
+  SET "status" = 'PLEDGED'
+  FROM "Collectives"
+  WHERE "Orders"."status" = 'PENDING'
+  AND "Orders"."PaymentMethodId" IS NOT NULL
+  AND "Collectives"."id" = "Orders"."CollectiveId"
+  AND "Collectives"."isActive" = FALSE
+  AND "Collectives"."isPledged" = TRUE`,
+    );
+  },
+
+  down: async (queryInterface, Sequelize) => {},
```

Comment:
Rollback is pretty simple for that one

```suggestion
  down: async (queryInterface, Sequelize) => {
    queryInterface.sequelize.query(`
      UPDATE "Orders"
      SET "status" = 'PENDING'
      WHERE "status" = 'PLEDGED'
    `);
  },
```